chroot_setup

```diff
@@ -0,0 +1,432 @@
+    # Unfortunately we are currently on v3.6.12, so we have to delete any
+    # directory that exists at that location.
+    if sys.version_info >= (3,8):
+      shutil.copytree(line, f"{args.chroot_directory}{line}", dirs_exist=True)
+    else:
+      if os.path.exists(f"{args.chroot_directory}{line}"):
+        shutil.rmtree(f"{args.chroot_directory}{line}")
+      shutil.copytree(line, f"{args.chroot_directory}{line}")
+
+# :chmod
+# Change the permissions on a file in the chroot.
+def chroot_chmod (line: str):
+  chmod_command = line.split()
+  assert len(chmod_command) == 2, "chmod command expects 2 arguments!"
+  mode = int(chmod_command[0], 8)
+  chmod_file = f"{args.chroot_directory}{chmod_command[1]}"
+  printf(f"os.chmod({chmod_file}, {mode})")
+  if not CHROOT_DEBUG:
+    os.chmod(chmod_file, mode)
+
+# :chmod_dir
+# Change the permissions on a directory and everything inside it in the chroot.
+def chroot_chmod_dir (line: str):
+  chmod_command = line.split()
+  assert len(chmod_command) == 2, "chmod command expects 2 arguments!"
+  mode = int(chmod_command[0], 8)
+  chmod_dir = f"{args.chroot_directory}{chmod_command[1]}"
+  for root, dirs, files in os.walk(f"{args.chroot_directory}{chmod_dir}"):
+    for dir in dirs:
+      print(f"os.chmod({os.path.join(root, dir)}, {int(mode, 8)})")
+      if not CHROOT_DEBUG:
+        os.chmod(os.path.join(root, dir), int(mode, 8))
+    for file in files:
+      print(f"os.chmod({os.path.join(root, file)}, {int(mode, 8)})")
+      if not CHROOT_DEBUG:
+        os.chmod(os.path.join(root, file), int(mode, 8))
+
+# :chown
+# Change ownership of a file in the chroot
+def chroot_chown (line: str):
+  chown_command = line.split()
+  assert len(chown_command) == 2, ":chown command expect 2 arguments!"
+  user = chown_command[0]
+  chown_file = f"{args.chroot_directory}{chown_command[1]}"
+  printf(f"os.chown({args.chroot_directory}{chown_file}, {user})")
+  if not CHROOT_DEBUG:
+    os.chown(f"{args.chroot_directory}{chown_file, user}")
+
+# :chown_dir
+# Change ownership of a directory and everything inside it in the chroot.
+def chroot_chown_dir (line: str):
+  chown_command = line.split()
+  assert len(chown_command) == 2, ":chown_dir command expect 2 arguments!"
+  user = chown_command[0]
+  chown_directory = f"{args.chroot_directory}{chown_command[1]}"
+  for root, dirs, files in os.walk(f"{args.chroot_directory}{chown_directory}"):
+    for dir in dirs:
+      printf(f"os.chown({os.path.join(root, dir)}, {user})")
+      if not CHROOT_DEBUG:
+        os.chown(os.path.join(root, dir), user)
+    for file in files:
+      printf(f"os.chown({os.path.join(root, file)}, {user})")
+      if not CHROOT_DEBUG:
+        os.chown(os.path.join(root, file), user)
+
+# :system
+# Executes a line from a .lst file using the /QOpenSys/usr/bin/system utility
+def chroot_system(line: str):
+  line = line.rstrip()
+  print(f"system -i {line}")
+  if not CHROOT_DEBUG:
+    os.system(f"system -i {line}")
```

Comment:
We should use [subprocess.run](https://docs.python.org/3/library/subprocess.html?ref=hackernoon.com#subprocess.run) instead of `os.system`

Reply:
Fixed this up in 3814d9b